src/platform/forms-system/src/js/components/ProgressButton.jsx

```diff
@@ -14,12 +14,18 @@ class ProgressButton extends React.Component {
 
   render() {
     const beforeText = this.props.beforeText ? (
-      <span className="button-icon">{this.props.beforeText} </span>
+      <span className="button-icon" aria-hidden="true">
+        {this.props.beforeText}
+        &nbsp;
```

Comment:
Adding a non-breaking space to prevent text & arrow getting separated by a new line. I've seen this in some older designs, but I wasn't able to duplicate it.